otx/algorithms/classification/adapters/mmcls/models/backbone/mobilenetv3.py

```diff
@@ -0,0 +1,385 @@
+        """Forward."""
+        if self.input_IN is not None:
+            x = self.input_IN(x)  # pylint: disable=not-callable
+
+        y = self.extract_features(x)
+        if return_featuremaps:
+            return y
+        # should be checked
+        return y
+
+
+class MobileNetV3(MobileNetV3Base):
+    """MobileNetV3."""
+
+    def __init__(self, cfgs, mode, IN_conv1=False, **kwargs):
+        super().__init__(**kwargs)
+        # setting of inverted residual blocks
+        self.cfgs = cfgs
+        assert mode in ["large", "small"]
+        # building first layer
+        input_channel = make_divisible(16 * self.width_mult, 8)
+        stride = 1 if self.in_size[0] < 100 else 2
+        layers = [conv_3x3_bn(3, input_channel, stride, IN_conv1)]
+        # building inverted residual blocks
+        block = InvertedResidual
+        flag = True
+        for k, t, c, use_se, use_hs, s in self.cfgs:
+            if (self.in_size[0] < 100) and (s == 2) and flag:
+                s = 1
+                flag = False
+            output_channel = make_divisible(c * self.width_mult, 8)
+            exp_size = make_divisible(input_channel * t, 8)
+            layers.append(block(input_channel, exp_size, output_channel, k, s, use_se, use_hs))
+            input_channel = output_channel
+        self.features = nn.Sequential(*layers)
+        # building last several layers
+        self.conv = conv_1x1_bn(input_channel, exp_size, self.loss)
+        output_channel = {"large": 1280, "small": 1024}
+        output_channel = (
+            make_divisible(output_channel[mode] * self.width_mult, 8) if self.width_mult > 1.0 else output_channel[mode]
+        )
+        self._initialize_weights()
+
+    def extract_features(self, x):
+        """Extract features."""
+
+        y = self.conv(self.features(x))
+        return y
+
+    def infer_head(self, x, skip_pool=False):
+        """Inference head."""
+
+        if not skip_pool:
+            glob_features = self._glob_feature_vector(x, self.pooling_type, reduce_dims=False)
+        else:
+            glob_features = x
+
+        logits = self.classifier(glob_features.view(x.shape[0], -1))
+        return glob_features, logits
+
+    def _initialize_weights(self):
+        """Initialize weights."""
+
+        for m in self.modules():
+            if isinstance(m, nn.Conv2d):
+                n = m.kernel_size[0] * m.kernel_size[1] * m.out_channels
+                m.weight.data.normal_(0, math.sqrt(2.0 / n))
+                if m.bias is not None:
+                    m.bias.data.zero_()
+            elif isinstance(m, nn.BatchNorm2d):
+                m.weight.data.fill_(1)
+                m.bias.data.zero_()
+            elif isinstance(m, nn.Linear):
+                n = m.weight.size(1)
+                m.weight.data.normal_(0, 0.01)
+                m.bias.data.zero_()
+
+
+@BACKBONES.register_module()
+class OTXMobileNetV3(MobileNetV3):
```

Comment:
Same here. Is it possible to inherit from torchvision? It will simplify the repo and avoid reimplementing. It might not be a scope of this PR but just something we can discuss. But if it is required by mmcv tasks then we can keep it.

Reply:
Please see the comment above.